public/app/features/dashboard/components/PanelEditor/VisualizationButton.tsx

```diff
@@ -42,7 +41,7 @@ export const VisualizationButton = ({ panel }: Props) => {
           imgSrc={plugin.meta.info.logos.small}
           isOpen={isVizPickerOpen}
           onClick={onToggleOpen}
-          aria-label={selectors.components.PanelEditor.toggleVizPicker}
```

Comment:
e2e tests rely on accessing this button through `components.PanelEditor.toggleVizPicker`, so please add it anywhere it's removed under [`data-testid` instead of `aria-label`](https://github.com/grafana/grafana/blob/main/contribute/style-guides/e2e.md#aria-labels-vs-data-testid) and check if the e2e tests are still failing like they are now 🙏

Reply:
Hello, thank you for your reply. I've made the changes accordingly. Let me know

Reply:
Hi 🙌 thank you for the quick reaction! The [e2e tests are now failing](https://github.com/grafana/grafana/blob/main/contribute/developer-guide.md#run-end-to-end-tests) because they expect aria-label, but they should be expecting data-testid. I'm sorry if this seems unrelated, but it's a part of the task of fixing the usability of aria-labels: removing them from use in e2e tests and [replacing that use](https://github.com/grafana/grafana/blob/main/contribute/style-guides/e2e.md#aria-labels-vs-data-testid) with data-testid.

Reply:
Can you direct to the file location? I am new to this tests

Reply:
@polibb I did the changes. Do let me know. Thanks!

Reply:
Hi, sorry, it is still failing 😕. Not sure why the betterer precommit hook is not running correctly, but would you run `yarn betterer` on this branch on your side, and commit the resulting files, if there are any? This might fix it.